src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```
@@ -62,13 +62,13 @@
       Text="Cross-OS native compilation is not supported. https://github.com/dotnet/corert/issues/5458" />
 
     <Error Condition="'$(DisableUnsupportedError)' != 'true' and !($(RuntimeIdentifier.EndsWith('x64')) or
```

We have `DisableUnsupportedError` to allow suppressing the error message. Adding the new platform to the list here is one of the last steps once everything is working.

---

Make sense.

---

General question. Does `DisableUnsupportedError` is `dotnet/runtime` property, or purely NativeAOT construct?

---

It is NativeAOT specific.